packages/gestalt/src/Button.jsdom.test.js

```diff
@@ -55,8 +55,8 @@ describe('Button', () => {
     render(<Button text="test" disabled ref={ref} />);
     expect(ref.current instanceof HTMLButtonElement).toEqual(true);
     expect(
-      ref.current instanceof HTMLButtonElement && ref.current?.tabIndex
```

Comment:
@AlbertCarreras could you verify that this is correct?

When I looked at a disabled `<Button />`, we only set the `disabled` attribute, we do not set the `tabIndex`:

1) Go to https://gestalt.netlify.app/Button
2) Toggle `Disable buttons`
3) Inspect the HTML in the Chrome devtools
4) No `tabIndex` is set on the button

![image](https://user-images.githubusercontent.com/127199/97900543-a111ec80-1cef-11eb-8911-324cd158db5c.png)

Reply:
The tabIndex gets to -1 if tabIndex is null 
![image](https://user-images.githubusercontent.com/10593890/97906045-d7ec0080-1cf7-11eb-93cd-314a5878b187.png)


![image](https://user-images.githubusercontent.com/10593890/97906078-e9cda380-1cf7-11eb-8478-29d410f0ecd1.png)